src/processTree.ts

```diff
@@ -4,10 +4,10 @@
  *
  *  Copied from https://github.com/microsoft/vscode-node-debug/blob/master/src/node/extension/processTree.ts
  *--------------------------------------------------------------------------------------------*/
-/* tslint:disable */
```

Comment:
keep this line to skip tslint on this file.

Reply:
Thank you! Done